ApplicationLibCode/ReservoirDataModel/Well/RigWellTargetCandidatesGenerator.cpp

```diff
@@ -747,10 +771,23 @@ RimRegularGridCase* RigWellTargetCandidatesGenerator::generateEnsembleCandidates
     {
         auto task = progInfo.task( "Accumulating results.", 1 );
 
-        accumulateResultsForSingleCase( *eclipseCase, *targetCase, resultNamesAndSamples, occupancy );
+        accumulateResultsForSingleCase( *eclipseCase, *targetCase, resultNamesAndSamples, occurrence );
     }
 
-    createResultVector( *targetCase, "OCCUPANCY", occupancy );
+    auto createFractionVector = []( const std::vector<int>& occurrence, int maxOccurrence ) -> std::vector<double>
```

Comment:
Rename `maxOccurrence` to `realizationCount` to make intention clearer.

Reply:
Fixed.